presets/4.3/tune/karate_race.txt

```diff
@@ -26,6 +26,11 @@ set gyro_lpf1_dyn_min_hz = 500
 set gyro_lpf1_dyn_max_hz = 1000
 set simplified_gyro_filter = OFF
 
+#$ OPTION BEGIN (UNCHECKED): Bosh (BMI) gyro - experimantal
```

Comment:
shouldn't this have the OSR4 cli command just to be sure... also ive had a number of people contact me and say they couldn't get the Bosch flying good so my solution was to kick the fixed stage2 gyro back to 250hz..

Reply:
I don't think all these Fcs are equal.. its the usual issue im seeing though not enough hardware filtering

Reply:
> I don't think all these Fcs are equal.. its the usual issue im seeing though not enough hardware filtering

so there are FCs with BMI gyro + OSR4 mode that requires MORE filtering that MPU6000? That's pretty wild for OSR4 mode.